grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/MedianVertexProgram.java

```diff
@@ -129,7 +129,7 @@ public void storeState(final Configuration configuration) {
     public void loadState(final Graph graph, final Configuration configuration) {
         super.loadState(graph, configuration);
         configuration.subset(RESOURCE_TYPE).getKeys().forEachRemaining(key ->
-                statisticsResourceLabelIds.add(LabelId.of(configuration.getInt(RESOURCE_TYPE + "." + key))));
+                statisticsResourceLabelIds.add((LabelId) configuration.getProperty(RESOURCE_TYPE + "." + key)));
```

Comment:
I like this more too. We put the actual `LabelId` in the config. I'm slightly concerned this won't work on a "real" cluster or something, so I'm going to wait for @duckofyork to confirm it's OK.